metricbeat/module/apache/status/status.go

```diff
@@ -12,6 +12,8 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/apache"
 )
 
+const AUTO_STRING  = "?auto"
```

Comment:
The string must be always present and there is no need to test it. It is simply the part of the URL for output for automatic processing like scripts...